setup.py

```diff
@@ -96,6 +96,7 @@ def read(f):
         'Chat: Gitter': 'https://gitter.im/aio-libs/Lobby',
         'CI: Azure Pipelines': 'https://dev.azure.com/aio-libs/aiohttp/_build',
         'Coverage: codecov': 'https://codecov.io/github/aio-libs/aiohttp',
+        'Docs: Changelog': 'https://docs.aiohttp.org/en/stable/changes.html',
```

Comment:
@scop I'd rather use this name as per https://github.com/pypa/warehouse/pull/7882/files#r422978485.